mkdocs/commands/serve.py

```diff
@@ -120,6 +121,9 @@ def builder():
 
         host, port = config['dev_addr']
 
+        if openbrowser:
+            webbrowser.open(config['site_url'])
```

Comment:
I have two concerns about this:

1. Why `site_url` and not `dev_addr`? I realize that when the dev server is used, we alter the `site_url` to match `dev_addr`, but is something were ever to go wrong with that, this would break things. Seems like we should just use `dev_addr` here.

2. Should't the browser be launched after the server is started, not before?